.github/workflows/lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Run lint and static analyis checks
+on:
+  pull_request:
+  workflow_call:
+
+jobs:
+  tardigradelint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone this git repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - name: Project Syntax Verification
+        run: make docker/run target=lint
+
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone this git repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```

Comment:
What is this git repository? This code is different then the previous so it makes me think its a different repository